SDGupdater/R/remove_superscripts.R

```diff
@@ -0,0 +1,115 @@
+#' @title Remove superscripts
+#'
+#' @description Remove superscripts, whether they are read as actual
+#'   superscripts or just as numbers (which are referred to as 'false
+#'   superscripts'). This function assumes that superscripts never follow:  
+#'   - a space  
+#'   - a number (so superscripts over 9 are not recognised)  
+#'   - a string consisting of just one other character  
+#'   - a 'word' consisting of both numbers and letters  
+#'   - a pound (�) or dollar ($) symbol.
+#'   - a string starting with a number or a single letter
+#'   If it becomes apparent that these scenarios need to be covered, the
+#'   function will need updating. 
+#'
+#' @param variable Character vector you want superscripts removed from.
+#'
+#' @return Character vector for all but `could_contain_superscripts`, which
+#'   returns a logical vector.
```

Comment:
For some reason this detail for `@return` doesn't appear in the function documentation if I view it through `?remove_superscripts` (but I can see it in the actual code). For the Return value it still just says "Character vector". I updated the package, but may still need a reset, I guess check if you're getting it from your end, it may just be me!

Reply:
I think this is because I hadn't updated the documentation. I have now done this and will push it to Github when I have addressed your other comments :)